internal/cli/config/config.go

```diff
@@ -248,7 +248,7 @@ func Builder() *cobra.Command {
 		Short: "Configure a profile to store access settings for your MongoDB deployment.",
 		Long: `Configure settings in a user profile.
 All settings are optional. You can specify settings individually by running: 
-  $ mongocli config set --help 
```

Comment:
[q] any reason for this change? it's an unwritten rule on our examples to separate text from code to add some indentation, you'll find this pattern around all our descriptions so if this change is needed we may want to do it all over the place and in a separate PR

Reply:
The reason is because this creates a build error in the generated RST. RST is extremely sensitive to indentation. Its reported about halfway down in the [build log](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=60a5690af1f6937450ed07f5), and here in VSCode:
![image](https://user-images.githubusercontent.com/47988788/119362523-a4f06680-bc61-11eb-9b06-a2e8d3277e76.png)

This is the only file that threw an error, but to be safe I searched through the rest of the repo and this is the only example I could find where indentation is used. I don't think we'd need to change this in other places, unless I'm looking in the wrong place?

Reply:
no, it looks like is a thing with `Description` and this is the only description with a command instead of an `Example`